Tests/Unit/EmogrifierTest.php

```diff
@@ -1763,4 +1763,27 @@ public function emptyMediaQueriesAreRemoved()
             $result
         );
     }
+
+    /**
+     * @test
+     */
+    public function multiLineMediaQuery()
+    {
+        $mediaQuery = "@media all and (max-width: 500px) {\r\n" .
```

Comment:
Please let's have two separate test cases: one with Windows line endings (\r\n) and one with Unix line endings (\n) so we know both cases work properly.